script.js

```diff
@@ -0,0 +1,199 @@
+    inputWrapper.innerHTML = `
+    <button id="aLotBtn">Obviously A LOT!</button>
+    <button id="alittleBtn">Just the regular dose</button>
+    `
+  } else if (oven === `Let's keep it cool`) {
+    inputWrapper.innerHTML = `
+    <button id="aLotBtn">Obviously A LOT!</button>
+    <button id="aLittleBtn">Just the regular dose</button>
+    `
+  } else {
+    inputWrapper.innerHTML = `
+    <button id="aLotBtn">Obviously A LOT!</button>
+    <button id="aLittleBtn">Just the regular dose</button>
+    `
+  }
+
+document
+.getElementById('aLotBtn')
+.addEventListener('click', () => nextQuestion(`obviously A LOT`))
+document
+.getElementById('aLittleBtn')
+.addEventListener('click', () => nextQuestion(`just the regular dose`))
+}
+
+const howMuchTime = (chocolate) => {
+  questionNumber++
+
+  botReply(`Ok, ${chocolate}, well noted. How much time do you have?`)
+
+  if (chocolate === `obviously A LOT`) {
+  inputWrapper.innerHTML = `
+    <button id="muchTimeBtn">All the time in the world, my schedule is cleared!</button>
+    <button id="littleTimeBtn">Not much, give me something quick & easy</button>
+  `
+} else {
+  inputWrapper.innerHTML = `
+  <button id="muchTimeBtn">All the time in the world, my schedule is cleared!</button>
+  <button id="littleTimeBtn">Not much, give me something quick & easy</button>
+  `
+}
+
+// unfortunately after this point my chatbot does not action and skips directly to the goodbye-message. What's missing?
+
+document
+  .getElementById('muchTimeBtn')
+  .addEventListener('click', () => nextQuestion('all the time in the world'))
+document
+  .getElementById('littleTimeBtn')
+  .addEventListener('click', () => nextQuestion('not much time'))
+}
+
+const suggestRecipe = (time) => {
+  questionNumber++
+
+// not sure if it works by sending 3 separate messages from the bot?
+
+  botReply(`If you have ${time} I suggest this recipe for you:`)
+
+const botReply = (oven, time) =>{
+  if (oven === `Let's keep it cool` && time === `not much time`) {  
+    showMessage (`img src="images/tiramisu.jpg" alt="Tiramisu" <br>
+    <a href =https://cherry-on-top.netlify.app/r_tiramisu.html' target="_blank">Tiramisu</a>`)} 
+  else if (oven === `Let's keep it cool` && time === `all the time in the world`) 
+    {showMessage (`img src="images/tobleronemousse.jpg" alt:"Toblerone Mousse" <br>
+    <a href =https://cherry-on-top.netlify.app/r_toblernemousse.html' target="_blank">Toblerone Mousse</a>`)} 
+  else if (oven === `Yes, that's hot!` && time === `not much time`)
+    {showMessage (`img src="images/lavacake.jpg" lat_"Chocolate Lava Cakes" <br>
+    <a href =https://cherry-on-top.netlify.app/r_lavacake.html' target="_blank">Chocolate Lava Cakes</a>`)} 
+  else if (userValue ==="Artwork")
+    {showMessage (`img src="images/Brownies.jpg" lat_"Brownies" <br>
+    <a href =https://cherry-on-top.netlify.app/r_brownies.html' target="_blank">Brownies</a>`)}
+  }
+  botReply(`Are you happy with your recipe?`
+  )
```

Comment:
Here you get an error about using `botReply` before initialization. I think you wanted to create a new function here - and if that's the case, you should call it something else, like `botRecommendation`

Reply:
Thinking more about it, you don't need this function. And you can't reach the oven variable in this function. One option is to declare a global variable that you can reach everywhere. Also, you have some small fixes to do in this block of code so I will make a list 😄
1. Declare a global variable on top of file `let ovenSelection = ""`
2. Change `oven` to `ovenSelection` in the marked block of code.
3. Add HTML tags around the `<img/>`s
4. Change `alt:` and `lat_` to `alt=`
5. Add an option if the user chooses "Yes, that's hot!" and "all the time in the world" to cover all options

I've done 2-4 here:
```
if (ovenSelection === `Let's keep it cool` && time === `not much time`) {  
    botReply(`<img src="images/tiramisu.jpg" alt="Tiramisu"/> <br>
    <a href =https://cherry-on-top.netlify.app/r_tiramisu.html' target="_blank">Tiramisu</a>`)
  } else if (ovenSelection === `Let's keep it cool` && time === `all the time in the world`) {
    botReply(`<img src="images/tobleronemousse.jpg" alt="Toblerone Mousse"/> <br>
    <a href =https://cherry-on-top.netlify.app/r_toblernemousse.html' target="_blank">Toblerone Mousse</a>`)
  } else if (ovenSelection === `Yes, that's hot!` && time === `not much time`) {
    botReply(`<img src="images/lavacake.jpg" alt="Chocolate Lava Cakes"/> <br>
    <a href =https://cherry-on-top.netlify.app/r_lavacake.html' target="_blank">Chocolate Lava Cakes</a>`)
  } 
```

Hopet his makes sense and work!